ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -9,7 +9,7 @@ if [[ -n $RESERVATION ]] ; then
   SERVICE_ACCOUNT=nrtrr
   PARTITION_DEFAULT=kjet
   PARTITION_FCST=kjet
-  PARTITION_GRAPHICS=kjet,xjet,tjet,ujet,vjet
+  PARTITION_GRAPHICS=kjet
```

Comment:
I set the graphics to run on kjet for the ensemble, and in the XML template, set them to also run in the reservations after go-ahead from David.

Reply:
If needed, can the graphics task be run on other Jet partitions? For the current RRFSE workflow, 00Z forecast will be done for the first 6 members for the 150 nodes. And then the next 3 members. I am a little concerned if the graphics task kicks off when the first 6 member forecast is done and therefore delaying the execution of member 7-9

Reply:
The graphics will wait until all the 00 hr wgrib2 tasks are complete before it kicks off, so shouldn't interfere with the forecast task node counts at all.